x/ratelimit/client/cli/query.go

```diff
@@ -128,3 +159,36 @@ func GetCmdQueryRateLimitsByChainId() *cobra.Command {
 
 	return cmd
 }
+
+// GetCmdQueryRateLimitsByChannelId return all rate limits that exist on the specified channel
```

Comment:
I think you can remove this CLI command since it's captured in `GetCmdQueryRateLimit`